pkg/log/multislogger/multislogger.go

```diff
@@ -8,6 +8,18 @@ import (
 	slogmulti "github.com/samber/slog-multi"
 )
 
+type contextKey string
+
+func (c contextKey) String() string {
+	return string(c)
+}
+
+const (
+	// KolideSessionIdKey this the also the saml session id
+	KolideSessionIdKey contextKey = "X-Kolide-Session"
+	SpanIdKey          contextKey = "span_id"
```

Comment:
This feels wrong. If these are context keys, they should probably be iotas. I don't think we should encode the header name here.

Reply:
no using iota, now the string of KolideSessionIdKey is `kolide_session_id`